src/interchange/src/protobuf.rs

```diff
@@ -104,68 +96,46 @@ impl DecodedDescriptors {
 pub struct Decoder {
     descriptors: DecodedDescriptors,
     packer: Row,
-    ccsr_client: Option<ccsr::Client>,
-    validated_schema_id: Option<i32>,
+    confluent_wire_format: bool,
 }
 
 impl Decoder {
     /// Constructs a decoder for a particular Protobuf message.
     pub fn new(
         descriptors: DecodedDescriptors,
-        schema_registry: Option<ccsr::ClientConfig>,
+        confluent_wire_format: bool,
```

Comment:
similarly, could update this to have two constructors but that seems less of a clear win